### ISSUE TITLE
Environment variable not detected since 1.7.4

### ISSUE THREAD

**Issue by skyborn8** [centrifugal/centrifugo#194]
Prior to version 1.7.4 centrifugo correctly used `CENTRIFUGO_SECRET` environment variable when present.
Starting with version 1.7.4, `no secret set in config` error is being issued.

Steps to reproduce:
**bash:**
```
CENTRIFUGO_SECRET=test centrifugo
```

**ruby:**
```ruby
  Centrifuge.secret = 'test'
  Centrifuge.publish('test', {})
 ```

@FZambia

**Comment by FZambia** [comment]
@skyborn8 hello, many thanks for a report!

Just pushed a fix in 3c27a7f35be21b083288744ce562080fec1efddc

The problem was setting env prefix in incorrect place, until new release it's possible to set prefix this way:

```
SECRET=test centrifugo
```

**Comment by skyborn8** [comment]
@FZambia awesome, thanks!

**Comment by FZambia** [comment]
Just released new version 1.7.6 with fix
